src/SelfServe/SqlX/SqlX.tsx

```diff
@@ -215,14 +228,22 @@ const calculateCost = (skuName: string, instanceCount: number): Description => {
       costPerHour += incrementalCost;
     }
 
+    if (costPerHour === 0) {
+      throw new Error("Cost per hour = 0");
+    }
```

Comment:
is this expected to happen?

Reply:
Happens if fetch prices api fails in any way